share/spice/get_events/get_events.js

```diff
@@ -28,21 +29,43 @@
                         return null;
                     }
                     return {
-                        url: buildUrl(item.id),
-                        name: item.name,
-                        description: DDG.strip_html(item.description),
-                        place: item.venue.name,
+                        data_front: {
+                            showPin: true,
+                            title: item.name,
+                            venue: item.venue,
+                            image: item.image_large_url,
+                            altSubtitle: formatSubtitleString(item),
+                            
+                            footer_content: Spice.get_events.foot_front,
+                            
+                            footLines: '4',
+                            titleClass: 'tile__title--3 tx--16 tx--bold mg--none',
+                            altSubClass: 'tx--13 tx-clr--grey'
+                        },
+                        data_back: {
+                            title: item.name,
+                            url: buildUrl(item.id),
+                            description: item.description ? DDG.strip_html(item.description) : 'No description available.',
+                            
+                            footer_content: Spice.get_events.foot_back,
+                            
+                            titleClass: 'tile__title--1 tx--16 tx--bold'
+                        },
                         city: item.venue.city,
-                        image: item.image_large_url,
-                        is_full_day: checkFullDay(item.start_date,item.end_date), 
-                        start_end: getStartEnd(item.start_date,item.end_date),
+                        place: item.venue.name,
                         lat: item.venue.lat,
                         lon: item.venue.lng
                     };
                 },
                 templates: {
```

Comment:
@bsstoner @moollaza how does this work without specifying sub-templates for front_content and back_content?